packages/inputs-tiny/.svgo.yml

```diff
@@ -0,0 +1,5 @@
+plugins:
```

Comment:
Depending on what gets merged first, you might have to update the svgo config to the new format, check this PR: https://github.com/goodhood-eu/goodhood/pull/136